sdk/src/wallet/account/operations/balance.rs

```diff
@@ -319,14 +222,95 @@ impl Account {
         Ok(())
     }
 
+    fn finish(
+        &self,
+        mut balance: Balance,
+        account_details: &AccountDetails,
+        network_id: u64,
+        total_rent_amount: u64,
+        total_native_tokens: NativeTokensBuilder,
+    ) -> Result<Balance> {
+        // for `available` get locked_outputs, sum outputs amount and subtract from total_amount
```

Comment:
It hasn't moved

Reply:
Was just confused because the section what this is about is so large and in between looks partially to be unrelated (which makes sense to have less iterations)